tmt/hardware.py

```diff
@@ -558,7 +560,7 @@ def _from_specification(
             if not isinstance(
                     value,
                     pint.Quantity):  # type: ignore[reportUnnecessaryIsInstance,unused-ignore]
-                value = pint.Quantity(value)
+                value = pint.Quantity(value)  # pyright: ignore[reportAssignmentType]
```

Comment:
Are changes to `hardware.py` needed if the file gets back on the "ignore` list?

Reply:
Oops, did not scroll down enough :sweat_smile: